rs/ic_os/fstrim_tool/BUILD.bazel

```diff
@@ -31,9 +31,8 @@ rust_library(
     aliases = ALIASES,
     crate_name = "ic_fstrim_tool",
     proc_macro_deps = MACRO_DEPENDENCIES,
-    visibility = [
-        "//rs:system-tests-pkg",
-    ],
+    # TODO: is this really needed?
+    visibility = ["//rs:system-tests-pkg"],
```

Comment:
This is probably not needed at all? The library is only used by the integration tests, so probably no external visibility is needed (I'm not a `bazel` expert though!)?

Reply:
Yes, I did see it is used in tests. 

However, when I looked at the exact code from the fs_trim tool I was wondering if even should be used in tests. The test depends on fstrim only to import a simple metrics type that it uses for deserializing some data.

Reply:
no action needed